src/ocm/components/clusterWizard/ClusterDetailsForm.tsx

```diff
@@ -66,6 +67,11 @@ const ClusterDetailsForm: React.FC<ClusterDetailsFormProps> = (props) => {
     [cluster, handleClusterCreate, handleClusterUpdate],
   );
 
+  const { search } = useLocation();
+  const params = new URLSearchParams(search);
+  const isArm = params.get('useArm') === 'true';
```

Comment:
Consider moving this logic into `ClusterDetailsService.getClusterDetailsInitialValues`